Automattic/gutenberg-everywhere
# Add/support content block

Migrated content embed block source code from https://github.com/Automattic/wp-calypso/tree/trunk/apps/happy-blocks/src/support-content . Block allows embedding:
- WordPress.com Guide (support) pages
- WordPress.com and other bbPress forum topics.

In addition to migration:
- Combined two blocks into one
- Add the ability to embed content blocks from other forums besides WordPress.com
- Visual fixes to address 118-gh/Automattic/lighthouse-forums

## Testing instructions

1. Build the plugin, sync to sandox.
2. Enable the `blocks-everywhere/support-content`
3. Create a new topic in the forum, in the editor press / and search for "Content embed" block.
4. Insert _public_ forum topic URL, for example, https://wordpress.com/forums/topic/request-a-quote-on-menu/, and verify its content has been embedded in the editor.
5. Save the topic, and verify embedded block was rendered in the topic view.
6. Verify embed is inserted by pasting the forum topic URL directly into the editor.
7. Repeat steps 3-5 for the Support page, for example https://wordpress.com/support/domains/change-a-domain/
